src/index.js

```diff
@@ -7,7 +7,7 @@ import CanvasRender from '@/components/render'
 import { ReaderContext, ReaderProvider } from '@/context'
 
 // Css
-import './css/styles.css'
+import './css/base.css'
```

Comment:
rename to `main.css` or `index.css` also use: `@/css/file.css`

Reply:
Changed 'base.css' to 'main.css', I also tried to change it using the format '@/css/main.css' but it breaks the build

Reply:
Ok, I'll look into it. its fine like that thanks :v: